src/app/zeko/da_layer/lib/batch.ml

```diff
@@ -7,9 +7,12 @@ module Stable = struct
   module V1 = struct
     type t =
       { source_ledger_hash : Ledger_hash.Stable.V1.t
+            (** Source ledger hash of the diff *)
       ; diff : (int * Account.Stable.V2.t) list
+            (** List of changed accounts with corresponding index in the ledger *)
       ; command_with_action_step_flags :
           (User_command.Stable.V2.t * bool list) option
+            (** Optionally add command with corresponding action steps to store the history *)
```

Comment:
Are there not cases where we might want to do multiple commands? Maybe not important though